lib/GLPI/Agent/Task/Inventory/Generic/Remote_Mgmt/MeshCentral.pm

```diff
@@ -0,0 +1,45 @@
+package GLPI::Agent::Task::Inventory::Generic::Remote_Mgmt::MeshCentral;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use English qw(-no_match_vars);
+
+use GLPI::Agent::Tools;
+
+sub isEnabled {
+    return OSNAME eq 'MSWin32' ?
+        canRun("C:\\Program Files\\Mesh Agent\\MeshAgent.exe") :
+        canRun('/usr/local/mesh_services/meshagent/meshagent');
+}
+
+sub _getNodeId {
+    return OSNAME eq 'MSWin32' ?
+    getFirstLine(
+        command => 'C:\\Program Files\\Mesh Agent\\MeshAgent.exe -nodeid') :
+    getFirstLine(
+        command => '/usr/local/mesh_services/meshagent/meshagent -nodeid');
```

Comment:
One question about it, inspecting on Linux Unit file says:
```
[Unit]
Description=meshagent background service
[Service]
WorkingDirectory=/usr/local/mesh_services/meshagent/
ExecStart=/usr/local/mesh_services/meshagent/meshagent --installedByUser=0
StandardOutput=null
Restart=on-failure
RestartSec=3
[Install]
WantedBy=multi-user.target
Alias=meshagent.service
```
So with this regex it can be _`Ex.*=(.*)\s\-`_ and good.

But i cant get an example based on registry, we need to acces here HKEY_LOCAL_MACHINE\SOFTWARE\Open Source\Mesh Agent and take NodeId, let me show you wich reg:
![image](https://user-images.githubusercontent.com/49277353/177397071-7709afba-0cde-4766-b3ec-79b4aca1c849.png)

Is there a registry module for Agent?